conf/application.conf

```diff
@@ -61,6 +61,7 @@ featureFlags {
   editNameEnabled = true
   fileUploadEnabled = false
   dvrEnabled = true
+  newAgentRelationshipJourneyEnabled = true
```

Comment:
can we default flag to false.

Reply:
I meant to delete that actually, the flag is in the dashboard

Reply:
and is defaulted to false